pico/tests/resources-pico/src/test/resources/expected/module-info.java._pico_

```diff
@@ -43,4 +44,4 @@ module io.helidon.pico.tests.pico {
     exports io.helidon.pico.tests.pico.provider;
     // Pico application - @io.helidon.common.Generated(value = "io.helidon.pico.tools.ApplicationCreatorDefault", trigger = "io.helidon.pico.tools.ApplicationCreatorDefault")
```

Comment:
Are we still using Pico? I thought we were supposed to rename this?

Reply:
It will be renamed (eventually before 4.x is released), not as part of this PR